src/engine/SCons/Taskmaster.py

```diff
@@ -792,7 +792,7 @@ def _find_next_ready_node(self):
         self.ready_exc = None
 
         T = self.trace
-        if T: T.write(SCons.Util.UnicodeType('\n') + self.trace_message('Looking for a node to evaluate'))
+        if T: T.write(str('\n') + self.trace_message('Looking for a node to evaluate'))
```

Comment:
don't think `'\n'` needs to be converted to `str`!

Reply:
Good call